[wallet connect] add executeSerializedMoveCall

gegaowp opened the pull request:
Add executeSerializedMoveCall to unblock capybara DApp,
the added function header is 
```
public executeSerializedMoveCall(txBytes: Uint8Array)
```

It now has the option from the Dapp console
<img width="1158" alt="Screen Shot 2022-07-07 at 12 47 40 PM" src="https://user-images.githubusercontent.com/106119108/177859591-09bebdb9-8f96-44fa-b1e7-f491fdaf7776.png">

pchrysochoidis reviewed (commented):
Looks good, if you have some time to revert [the renaming of transaction to tx](https://github.com/MystenLabs/sui/pull/3072/files#r916609922) and consider [this](https://github.com/MystenLabs/sui/pull/3026#discussion_r916596104) instead of introducing the txBytes field let me know, otherwise I will approve it as is

pchrysochoidis commented on wallet/src/dapp-interface/DAppInterface.ts:
```diff
@@ -86,11 +86,21 @@ export class DAppInterface {
     }
 
-    public executeMoveCall(transaction: MoveCallTransaction) {
+    public executeMoveCall(tx: MoveCallTransaction) {
```
Why did we rename this? Leaving it as `transaction` we can avoid changing 

```
{
                type: 'execute-transaction-request',
                transaction,
}
```
to

```
{
                type: 'execute-transaction-request',
                transaction: tx,
}
```

gegaowp commented:
addressed comments 1. revert the transaction naming change 2. change TransactionRequest type definition per Pavlos's comment
Also added contents to show on signing page for serialized txn.

pchrysochoidis reviewed (approved):
Thanks @gegaowp for addressing the comments. Although, I thinκ we don't need to introduce the `txBytes` and use the `type` everywhere from the messages to handling. But this looks good as is and we can update it later. 👍
